tutorials/tour/_posts/2017-02-13-singleton-objects.md

```diff
@@ -10,65 +10,111 @@ num: 13
 
 next-page: xml-processing
 previous-page: pattern-matching
+assumed-knowledge: classes, methods, private-methods, packages
 ---
 
-Methods and values that aren't associated with individual instances of a [class](classes.html) belong in *singleton objects*, denoted by using the keyword `object` instead of `class`.
+A singleton object is an instance of a new class. There is exactly one instance of each singleton object. They do not have constructors so they cannot be instantiated.
 
+# Defining a singleton object
+Like with classes, the simplest form of an object is the keyword `object` and an identifier:
+```tut
```

Comment:
we should be marking code blocks `tut` whenever possible, so PR validation checks that the code compiles. see https://github.com/scala/scala.github.com/issues/677